phpunit.xml

```diff
@@ -28,7 +28,7 @@
             <file phpVersion="7.4.0" phpVersionOperator=">=">./tests/ReflectionClosure5Test.php</file>
         </testsuite>
         <testsuite name="ReflectionClosure6">
-            <file phpVersion="8.0.0-x" phpVersionOperator=">=">./tests/ReflectionClosure6Test.php</file>
```

Comment:
I specifically changed this because `8.0.0-dev` is not greater than `8.0.0`. The file is being completely ignored. On my local declaring it as `8.0.0-x` is correctly running the file

Reply:
`8.0.0` should work. It works for mockery's test suite. https://github.com/mockery/mockery/blob/master/phpunit.xml.dist

Reply:
Apparently what makes it work for Mockery is `<directory suffix="Test.php">./tests</directory>` and not the phpVersion comparison line.

Reply:
See https://3v4l.org/VIgRZ

Reply:
Hmmm. What if you wrote `8.0.0-dev`, there?

Reply:
Done